apps/trading/e2e/tests/liquidity_provision/test_liquidity_provision.py

```diff
@@ -46,7 +60,35 @@ def test_liquidity_provision_amendment(continuous_market, vega: VegaService, pag
     )
     next_epoch(vega=vega)
     page.reload()
+    expect(page.get_by_test_id("supplied-stake")).to_have_text("Supplied stake1.00001 tDAI")
+    expect(page.get_by_test_id("liquidity-supplied")).to_have_text("Liquidity supplied 17.16%")
+    row = page.get_by_test_id("tab-myLP").locator(".ag-center-cols-container .ag-row").first
+    expect(row).to_contain_text(
+        "Active"
+    )
+@pytest.mark.skip("Waiting for the ability to cancel LP")
+@pytest.mark.usefixtures("page", "auth", "risk_accepted")
+def test_liquidity_provision_inactive(continuous_market, vega: VegaService, page: Page):
+    # TODO Refactor asserting the grid
+    page.goto(f"/#/liquidity/{continuous_market}")
+    page.get_by_test_id("manage-vega-wallet").click()
+    #TODO Rename "mm" to "marketMaker" so that we don't have to specify where to click when switching wallets
+    # Currently the default click will click the middle of the element which will click the copy wallet key button
+    page.locator('[role="menuitemradio"] >> .mr-2.uppercase').nth(1).click(position={ "x": 0, "y": 0}, force=True)
```

Comment:
Can we change this so we do not have co-ordinates? Before with the copy button there was a genuine element that did not include the copy button. This may not be true for this particular element so I will let you judge.

Reply:
Created a helper